host.yaml

```diff
@@ -154,4 +154,7 @@ smw_options:
   # Set this to false to never autostart a rom (such as after patching)
   # True for operating system default program
   # Alternatively, a path to a program to open the .sfc file with
-  rom_start: true
```

Comment:
Was this an intentional removal? Seems like this could impact smw?

Reply:
I didn't make this change, it came from merging in changes from main. Checking the main repository, it's missing there as well, so I'm guessing it's okay?

Reply:
Oh, weird that it's missing there and still shows that this is a removal. Yeah, if it's missing there I would imagine this isn't an issue, but I'm not sure.